tests/test-projects/live-reloading/schemas/syntax-error.js

```diff
@@ -1,13 +1,13 @@
-import { graphQLSchemaExtension, list } from '@keystone-6/core';
-import { text } from '@keystone-6/core/fields';
+import { graphQLSchemaExtension, list } from '@keystone-6/core'
+import { text } from '@keystone-6/core/fields'
 
-expor const lists = {
+export const lists = {
```

Comment:
This is an intentional syntax error, in any event

Reply:
Oooh, ok 😂 prettier failed here. Will revert and igonore